src/nft/components/profile/list/FloatingBanner.tsx

```diff
@@ -0,0 +1,112 @@
+import { buttonHoverState } from 'components/Common'
+import { getTotalEthValue } from 'nft/components/bag/profile/utils'
+import { useSellAsset } from 'nft/hooks'
+import { Listing, WalletAsset } from 'nft/types'
+import { formatUsdPrice } from 'nft/utils/currency'
+import { fetchPrice } from 'nft/utils/fetchPrice'
+import { useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const FloatingBannerGradientContainer = styled.div`
+  display: none;
+  position: fixed;
+
+  justify-content: center;
+  bottom: 0;
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, #000000 100%);
+  width: calc(100% - 360px);
+  left: calc((100% - 360px) / 2);
+  transform: translateX(-50%);
+  padding-top: 48px;
+  padding-bottom: 48px;
+
+  @media only screen and (min-width: ${({ theme }) => `${theme.breakpoint.lg}px`}) {
+    display: flex;
+  }
+`
+
+const FloatingBannerContainer = styled.div`
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  border-radius: 20px;
+  display: flex;
+  width: calc(100% - 100px);
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px 48px;
+`
+
+const Content = styled.div`
+  display: flex;
+  gap: 16px;
+  align-items: center;
+`
+
+const ListButton = styled.button`
```

Comment:
There is a component that exists already called `ListingButton` that has a lot of the logic around validation built in. I would suggest adding and modifying that component as opposed to making one from scratch. For example, this new button doesn't check if duration is valid.
![Screen Shot 2022-12-09 at 08 06 04 ](https://user-images.githubusercontent.com/11512321/206743592-980cb6e1-a119-4c8a-863a-3540f6735d01.png)

Reply:
it's possible the logic will change and it complicates the pr significantly.  It was left out intentionally